src/summon/browzine-summon-adapter.js

```diff
@@ -383,7 +383,7 @@ browzine.summon = (function() {
     var featureEnabled = false;
     var config = browzine.articleLinkEnabled;
 
-    if (config === true) {
+    if (typeof config === "undefined" || config === null || config === true) {
```

Comment:
Works for me 👍 

We could also just change it to check if `config === false` perhaps? But no biggie, this definitely gets the job done, and should be sure not to break the existing installs.

Reply:
> We could also just change it to check if config === false perhaps?

One would think, but alas JavaScript...
![image](https://user-images.githubusercontent.com/376230/102637226-adc59500-4123-11eb-9862-37689ce10743.png)

The current setup helps us account for when the `articleLinkEnabled` property doesn't exist at all. e.g `undefined` mainly, but possibly even a `null` value.

Reply:
JavaScript!
![iu-16](https://user-images.githubusercontent.com/1091559/102637686-faa56d80-411b-11eb-992d-3d1cfb370e43.gif)

Thanks for ensuring we didn't ship my wrong idea 😄